doc/ref_model/chapters/chapter03.rst

```diff
@@ -548,35 +540,32 @@ The creation of the SFC might include design/preparation phase as:
 
 1. Creation of service functions of SFC.
 
--  The flow of steps to enable the SFC creation can be as follows:
-
-   a. SFC orchestrator creates the SFs with help of VNF MANO or CNF MANO.
-   b. SFC Renderer attaches the SFC aware interfaces at SFs to enable Service plane
-   c. NFVO boots up the relevant SF configurations at SF.
-
+   - The flow of steps to enable the SFC creation can be as follows:
+     a. SFC orchestrator creates the SFs with help of VNF MANO or CNF MANO.
```

Comment:
Leave an empty line before the a-b-c list.

Reply:
@CsatariGergely done.